msteams-platform/get-started/deploy-csharp-app-studio.md

```diff
@@ -19,33 +19,33 @@ Complete the following steps to update the app package:
 
 1. To install App Studio in Teams, select the **Apps** icon at the bottom of the left-hand bar, and search for **App Studio**:
 
-    <img  width="450px" alt="Finding App Studio in the Store View" src="~/assets/images/get-started/searchforAppStudio.png"/>
+    :::image type="content" source="~/assets/images/get-started/searchforAppStudio.png" alt-text="Finding app studio in the store view.":::
 
 1. Select the **App Studio** tile and choose **Install**. The App Studio is installed:
 
-    <img  width="450px" alt="Installing App Studio" src="~/assets/images/get-started/InstallingAppStudio.png"/>
+    :::image type="content" source="~/assets/images/get-started/InstallingAppStudio.png" alt-text="Installing app studio.":::
 
 1. To create the app package for your Teams app, select the **Manifest editor** tab in **App Studio**:
 
-    <img  width="450px" alt="App Studio" src="~/assets/images/get-started/AppStudio.png"/>
+    :::image type="content" source="~/assets/images/get-started/AppStudio.png" alt-text="The screenshot describes on how to select manifest editor to create app package.":::
 
     The sample comes with its own manifest and is designed to build an app package when the project is built. The manifest.json file can be located in Visual Studio in Manifest under ```Microsoft.Teams.Samples.HelloWorld.Web```.
 
      In Visual Studio, the manifest.json file is located in under **Manifest** in `Microsoft.Teams.Samples.HelloWorld.Web`. This step is described by the following image:  
 
-    <img  width="450px" alt="Build the app package on .NET with Visual Studio" src="~/assets/images/get-started/app-package-on-.NET-with-Visual-Studio.png"/>
+    :::image type="content" source="~/assets/images/get-started/app-package-on-.NET-with-Visual-Studio.png" alt-text="The screenshot describes how to build the app package on .NET with visual studio.":::
```

Comment:
@v-habibaumme , what are all these characters doing in the image file name? And what's with the mixed casing?

Reply:
resolved